angular-workspace/projects/example-client-app/src/app/customapp/customapp.component.ts

```diff
@@ -55,11 +57,23 @@ export class CustomAppComponent {
 5. Absolute link: <https://nimble.ni.dev/>
 `;
 
+    public editorMarkdownString = `Supported rich text formatting options:
```

Comment:
nit: Just curious. Any specific reason to maintain it in a separate variable? Why don't we retain the variable as common. i.e., `markdownString`

Reply:
The only difference is the absolute link we have in the viewer is removed in the editor sample markdown string. But as we discussed offline, anyways we will be supporting absolute links in the editor. So I am using a same string for both the components.